test/Common/TestHelpers.cs

```diff
@@ -640,4 +641,35 @@ public string Resolve(string name)
             }
         }
     }
+
+    /// <summary>
+    /// Test helper implementation of INameResolver interface.
+    /// </summary>
+    public class SimpleNameResolver : INameResolver
```

Comment:
Public classes should be placed in their own file. The only exception for this should be inner classes.

Reply:
Great point! I will be careful in future.